src/soup.ml

```diff
@@ -1,6 +1,8 @@
 (* This file is part of Lambda Soup, released under the MIT license. See
    LICENSE.md for details, or visit https://github.com/aantron/lambdasoup. *)
 
+exception Selector_parse_error of string
```

Comment:
Can this be shortened to just `Parse_error`? IIRC Lambda Soup never parses anything except selectors.

Reply:
Sorry, I forgot to push it! Fixed now.

Reply:
GitHub said you marked this as resolved, but your reply addresses my *other* comment :)

Reply:
Sorry for the confusion! That's a good point, since HTML parsing is done by Markup, even if starts raising exceptions at some point, they will be in a different module.
I've renamed it.